### DISCUSSION TITLE
Nothing happens after starting testbed.exe

### DISCUSSION THREAD

**Discussion by vikista** [NVlabs/instant-ngp#400] [Q&A]
Hello after building everything, executable just doesnt start,
Windows 11
3070
11800h

**Comment by leventt** [comment]
Windows Defender is a bit strict nowadays and may be blocking that exe file if you are double clicking it.
You can try running it via cmd to also copy any logs here for more help.
Make sure you run the testbed.exe file from the root folder of the repo (where you cloned it)
